modules/library/src/main/scala/zio/elasticsearch/ElasticHighlight.scala

```diff
@@ -0,0 +1,14 @@
+package zio.elasticsearch
+
+import zio.Chunk
+import zio.elasticsearch.highlighting.{Highlights, HighlightField}
+import zio.json.ast.Json
+
+object ElasticHighlight {
+
+  def highlight[S](field: Field[S, _], fieldConfig: Map[String, Json]): Highlights =
```

Comment:
Can we add a default `Map.empty` here as well?

Reply:
`multiple overloaded alternatives of method highlight define default arguments.`

We have this issue and should talk about possible solutions.